pkgs/by-name/aw/aws-nuke/package.nix

```diff
@@ -33,6 +33,7 @@ buildGoModule rec {
   ldflags = [
     "-s"
     "-w"
+    "-extldflags=\"-static\""
```

Comment:
Any particular reason for adding this? Does it not work properly when dynamically linked?